src/plugins/cosmos/components/modals/Staking/views/Stake.tsx

```diff
@@ -96,9 +119,28 @@ export const Stake = ({ assetId, apr }: StakeProps) => {
       .times(_percent)
       .dp(asset.precision, BigNumber.ROUND_DOWN)
     const fiatAmount = bnOrZero(cryptoAmount).times(marketData.price)
+    const fiatAmountMinusTxFee = fiatAmount.minus(averageTxFee?.fiatFee ?? '0')
```

Comment:
bnorzero?

Reply:
Sweet, that's cleaner yeah, removed it in all lines of this diff